source/includes/api/performance/_allocated_claims.md

```diff
@@ -78,3 +78,6 @@ Eg:
 Attribute | Type | Info
 --------- | ---- | ----
 some\_custom\_field | `custom field type` | A custom answer for this field
+
+These data fields can be obtained from the following endpoint...
+[`GET /api/v2/performance/promotions/1/data_fields`](/#list-all-data-fields)
```

Comment:
1. Don't need the `/` in the URL?
2. Feels a bit redundant having the link here, transform "GET List Data Fields" in the paragraph above to the link?